### ISSUE TITLE
Version 2023.5.1 blocked by virus/anti malware scanner

### ISSUE THREAD

**Issue by Suffekat** [rgriebl/brickstore#740]
**Describe the bug**
The BrickStore application is terminated and quarentied by the anti virus/ anti malware scanner the moment you try to download the store inventory. Starting up the application is fine, but the moment you try to do something with inventory the virus scanner kicks in and quarentines the file "BrickStore.exe". The reason given is that there is "unauthorized file encryption"

Virus scanner is Trend Micro Apex One, agent version 14.0.12571
Older versions of Brickstore do not have this issue.

The virus scanner is managed corporatly and I cannot change scna settings or something like that.
Please find detailed screenshots attached.

**To Reproduce**
1. Use virus Trend Micro virus scanner
2. Open BrickStore
3. Download BrickLink Store inventory
5. Brickstore is terminated immediatly

**Expected behavior**
No interference from anti-virus scanner as it was in previous versions of BrickStore

**Screenshots**
See Attached

**System information**
### BrickStore 2023.5.1 (build: 1049)

* **brickstore.ldraw**: 19 Mar 2023 12:29:57 +0100
* **brickstore.locale**: nl
* **build.arch**: x86_64
* **build.compiler**: Microsoft Visual-C++2019 (192930148)
* **build.date**: 2023-05-29T16:12:29.000
* **build.host**: fv-az171-567
* **build.number**: 1049
* **build.qt.version**: 6.5.1
* **hw.cpu**: Intel(R) Core(TM) i7-8665U CPU @ 1.90GHz
* **hw.cpu.arch**: x86_64
* **hw.gpu**: Intel(R) UHD Graphics 620
* **hw.gpu.arch**: intel
* **hw.memory.gb**: 15.8
* **os.arch**: x86_64
* **os.productname**: Windows 10 Version 22H2
* **qt.debug**: false
* **qt.platformname**: windows
* **qt.version**: 6.5.1
* **windows.tabletmode**: false

**Additional context**
![Trend info](https://github.com/rgriebl/brickstore/assets/136059577/e6860c70-758e-4c7a-96b1-7ec115608514)
![trend log file](https://github.com/rgriebl/brickstore/assets/136059577/8d6844ae-5acc-4944-843f-615568287922)

**Comment by rgriebl** [comment]
Duplicate of #684 and #695
TL;DR: BrickStore is clean, Trend Micro sucks. There is nothing I can do, sorry.
